apps/sms/views/conversation/js/conversation.js

```diff
@@ -1203,13 +1211,15 @@ var ConversationView = {
 
     // We're waiting for the keyboard to disappear before animating back
     return this._ensureKeyboardIsHidden().then(function() {
+      var isThread = Navigation.isCurrentPanel('thread');
```

Comment:
nit: "isThread" => "inConversation" ? (we'll eventually move away from the term "thread" so better start now)